webapp/channels/src/components/post_edit_history/edited_post_item/edited_post_item.tsx

```diff
@@ -215,16 +229,14 @@ const EditedPostItem = ({post, isCurrent = false, postCurrentVersion, theme, act
                 >
                     <div
                         className='edit-post-history__title__container'
-                        aria-hidden='true'
                     >
                         <div className='edit-post-history__date__badge__container'>
-                            <IconButton
-                                size={'sm'}
-                                icon={open ? 'chevron-down' : 'chevron-right'}
-                                compact={true}
+                            <button
                                 aria-label='Toggle to see an old message.'
                                 className='edit-post-history__icon__button toggleCollapseButton'
```

Comment:
why can we just use the standard button classes from `webapp/channels/src/sass/components/_buttons.scss`